ofono/modem.go

```diff
@@ -390,7 +469,7 @@ func (modem *Modem) GetMMSContexts(preferredContext dbus.ObjectPath) (mmsContext
 	}
 
 	for _, context := range contexts {
-		if (context.isTypeInternet() && context.isActive() && context.hasMessageCenter()) || context.isTypeMMS() {
+		if (context.isTypeInternet() && context.hasMessageCenter()) || context.isTypeMMS() {
```

Comment:
It did not, but performed a couple of formatting changes, so I have resubmitted anyway :)